src/neptune_tensorflow_keras/impl/version.py

```diff
@@ -13,6 +14,15 @@
         version,
     )
 
+if not (find_spec("neptune") or find_spec("neptune-client")):
+    msg = """
+            neptune package not found.
+
+            Install neptune package by running
+                `pip install neptune`
+                    """
```

Comment:
@normandy7 I'll probably need your help on that part :)

Reply:
Hopefully the formatting checks out; let me know if there's an issue :)